File: Android/EmotionsApp/EmotionsApp/Models/DTOs/Content/Note/NoteResponseDTO.swift
//
//  NoteResponseDTO.swift
//  EmotionsApp
//
//  Created by Муса Зарифянов on 09.11.2025.
//

import Foundation

struct NoteResponseDTO: Codable, Sendable {
    var id: Int64
    var albumId: Int64
    var title: String
    var content: String
    var createdAt: Date
}

File: Android/app/src/main/java/com/example/emotionapp/data/dtos/NoteResponseDTO.kt
package com.example.emotionapp.data.dtos

import kotlinx.serialization.Serializable
import java.util.Date

@Serializable
data class NoteCreateDTO(
    val albumId: Long,
    val title: String,
    val content: String
)

@Serializable
data class NoteResponseDTO(
    val id: Long,
    val albumId: Long,
    val title: String,
    val content: String,
    val createdAt: String
)

@Serializable
data class NoteShortDTO(
    val id: Long,
    val title: String,
    val createdAt: String
)

@Serializable
data class NoteUpdateDTO(
    val title: String? = null,
    val content: String? = null
)
